Improved Darkmode

lxndrblz opened the pull request.

ericswpark commented:
Will this also fix the issue with the code highlighting when in dark mode? Right now it's hard to see highlighted code when the site is switched to dark mode:

<img width="528" alt="image" src="https://user-images.githubusercontent.com/13326074/112761048-3bbeef00-9034-11eb-8d50-7f6cf7df99a1.png">

lxndrblz commented:
@ericswpark  That's still something I need to fix, but it is on my todo list.

lxndrblz commented:
@ericswpark Feel free to check out my latest adjustments. I've added  a bit of JavaScript, that checks if the dark mode was set based on a css query. Once the users changes the mode, that will have precedence, because otherwise it would result in some super weird switching.

I've also added back the faded, which you have suggested. Personally, I think it doesn't look that great (especially from light to dark), but I am curious to see what you think.
